pkg/services/resource.go

```diff
@@ -114,7 +115,12 @@ func (s *sqlResourceService) Replace(ctx context.Context, resource *api.Resource
 	}
 
 	found.Manifest = resource.Manifest
-	updated, err := s.resourceDao.Replace(ctx, found)
+	if resource.Version > found.Version {
```

Comment:
I think the version should only be maintained by maestro server, we should not allow to update this by client, this part has implemented by https://github.com/openshift-online/maestro/pull/7

Reply:
will rebase this code once #7 is merged.

Reply:
rebased code and added version check before updating resource, see: https://github.com/openshift-online/maestro/pull/12/files#diff-e771a4841dd0e3799d1956f0b8298ac034f67984e5e7f075e278b74fbd754847